joomla/joomla-cms
# Search Tools (from JLayouts) implemented in com_menu (menus view)

#### Description

In move to normalize the search/filter/order/limit bar (aka `searchtools`) in all administrator list views, and as other PR's have made for other list component/views, this PR adds JLayouts `searchtools` to com_menu `menus` view and replacing the old HTML code.
With that it also adds an option for ordering to com_menu menus view.

A PHP notice in hathor was corrected too.
##### Isis template
###### Before patch

![image](https://cloud.githubusercontent.com/assets/9630530/12128514/b3e2781a-b3f5-11e5-9656-046b4ea4dae1.png)
###### After patch

![image](https://cloud.githubusercontent.com/assets/9630530/12128540/f3937e6e-b3f5-11e5-96e3-a528b2305fe9.png)
##### Hathor template
###### Before patch

![image](https://cloud.githubusercontent.com/assets/9630530/12128633/6b5df924-b3f6-11e5-84df-693236415623.png)
###### After patch

![image](https://cloud.githubusercontent.com/assets/9630530/12128602/3a9fdd98-b3f6-11e5-8fa0-f17f5b97a986.png)
#### How to test
1. Go to com_menu menus view (the one with the menutypes, not menu items view).
2. Create some menus (at least one of them without linked modules)
3. Apply the patch
4. Test the Search Tools (Search field, ordering and limit).
5. View hathor template to and check the php notice is not there now.
#### Observations

First time using a part of JLayouts so suggestions and code revisions are welcome.
#### Tags

administrator, menus, search tools, jlayouts, list views, normalization
